Sources/RunCatLocalization/RCLLanguage.swift

```diff
@@ -25,6 +26,7 @@ public enum RCLLanguage: String, Sendable, Identifiable {
     public static let allCases: [RCLLanguage] = [
         .english,
         .japanese,
-        .korean
+        .korean,
+        .simplifiedChinese
```

Comment:
Please add a comma at the end of line.

```suggestion
        .simplifiedChinese,
```